CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.21.0-rc.0 / 2020-08-26
+
+* [CHANGE] Promtool: Add tsdb commands as promtool tsdb subcommand. #6088
+* [FEATURE] Docker Swarm SD: Support tasks and service without published ports. #7686
+* [FEATURE] Eureka SD: New service discovery. #3369
+* [FEATURE] Hetzner SD: New service discovery. #7822
+* [FEATURE] Kubernetes SD: Support Kubernetes EndpointSlices. #6838
+* [ENHANCEMENT] Support composite durations in PromQL, config and UI, e.g. 1h30m. #7713 #7833
+* [ENHANCEMENT] DNS SD: Add SRV record target and port meta labels. #7678
+* [ENHANCEMENT] PromQL: Reduce the amount of data queried by remote read when a subquery has an offset. #7667
+* [ENHANCEMENT] Promtool: add --time option to query instant command. #7829
+* [ENHANCEMENT] Rules: Label rule_group_iterations metric with group name. #7823
+* [ENHANCEMENT] Scrape: Add per scrape-config targets limit. #7554
+* [ENHANCEMENT] Scrape: Do not stop scrapes in progress during reload. #7752
+* [ENHANCEMENT] UI: Change dropdown hover color to white. #5593
+* [ENHANCEMENT] UI: Move "remove graph" closer to query area. #5990
+* [BUGFIX] Disable HTTP2 because of concerns with the go HTTP2 client. #7588 #7701
+* [BUGFIX] PromQL: Drop metric name in bool comparison between two instant vectors. #7819
+* [BUGFIX] PromQL: query_log_file path is now relative to the config file. #7701
+* [BUGFIX] Rules: Detect extra fields in rule files. #7767
+* [BUGFIX] Rules: Disallow overwriting the metric name in the `labels` section of recording rules. #7787
+* [BUGFIX] Rules: Keep evaluation timestamp across reloads. #7775
+* [BUGFIX] TSDB: Fix 'chunks.HeadReadWriter: maxt of the files are not set' error. #7856
+* [BUGFIX] Triton SD: Fix a panic when triton_sd_config is nil. #7671 #7701
```

Comment:
I'm not sure why 7701 is listed here.

Reply:
It removes the need for #7671, it would have fixed the bug too

Reply:
That's not really relevant here, 7671 was the first change to fix this issue.

Reply:
ack